tools/maxquant/maxquant.xml

```diff
@@ -1012,6 +1012,15 @@ This tool is a wrapper for MaxQuant v@VERSION@. The current version of the wrapp
                     File3       f3          E1            (fraction not an integer)
                     File4       1                         (missing experiment)
                                                           (File5 missing)
+        - However, “.raw” (Thermo) is considered a part of the filename:
+
+                ::
+
+                    Name         Fraction    Experiment   PTM
+                    File1.raw        1          E1       False
+                    File2.raw        2          E1       False
+                    File3.raw        3          E1       False
+                    File4.raw        4          E2       False
```

Comment:
just realised that the fraction for a new experiment would start at 1 again. Name: File4.raw; Fraction: 1; Experiment E2, PTM: False. Maybe to make it less confusing add File5.raw,2,E2, False and File6.raw,3,E2,False